README.md

```diff
@@ -1,2 +1,155 @@
-# Hello-Microverse
-This simple project shows how to set up a "Hello Microverse" repository to demonstrate Linter configuration and GitHub action.
+<a name="readme-top"></a>
+
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
Kindly take away the Microverse logo from your readme as you are required to personalize the readme file😉. alternatively, you can add your personal brand logo😉.